MVCapplication/src/main/java/com/softserve/if072/mvcapp/service/PdfCreatorService.java

```diff
@@ -49,6 +53,8 @@ public PdfCreatorService(UserService userService,
     private Font LARGE = new Font(bf, 18, Font.BOLD);
     private Font SMALL_BOLD = new Font(bf, 12, Font.BOLD);
     private Font SMALL = new Font(bf, 12);
+    private Font SMALL_RED = new Font(bf, 12, Font.NORMAL, new Color(180, 0x00, 0x00));
+    private Font SMALL_GREEN = new Font(bf, 12, Font.NORMAL, new Color(26, 193, 25));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "SMALL_GREEN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00116)